tests/runtime-tests/README.md

```diff
@@ -0,0 +1,27 @@
+# Runtime Tests
+
+The runtime tests ensure that Spin can properly run applications. 
+
+For the purposes of these tests, an "application" is a collection of the following things:
+* A Spin compliant WebAssembly binary
+* A spin.toml manifest
+* A list of arguments to be passed to `spin up`
+* A collection of a files needed for running `spin up` (e.g., a migration file for passing to the `--sqlite` argument)
```

Comment:
would we also need plugins such as `js2wasm` (or `py2wasm`) to run tests for JS/TS/Python etc?

Reply:
Added clarification in the README - we are not testing that js2wasm produces the correct WebAssembly components. Tests for that should either remain as end to end tests or IMO they should just be tests in the plugin repo themselves.

Reply:
this makes sense. thanks Ryan